.travis.yml

```diff
@@ -21,6 +22,8 @@ matrix:
           env: TOXENV=py35
         - python: pypy
           env: TOXENV=pypy
+        - python: 3.5
```

Comment:
I'd like a 2.7 as well here. I think I'm convinced it's not super important to have the entire set of 2.6-3.5.